### ISSUE TITLE
Could you provide validation accuracy for each class when you train the VideoTag model?

### ISSUE THREAD

**Issue by yushuinanrong** [PaddlePaddle/PaddleVideo#233]
Thanks for sharing this amazing codebase and pre-trained models!
I wonder if you could share the validation/testing accuracy for each of the 3396 classes when you train the VideoTag model.
